data/systemd/snapd.manage

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
```

Comment:
Nitpick idea: `snapd.entrypoint` instead of manage.

Reply:
I like your suggestion from irc: `snapd.run-from-snap`